src/UserService.ts

```diff
@@ -53,6 +53,52 @@ export type SubscriptionInfo = {
   trialEndsAt?: Date | null;
 };
 
+export type Address = {
+  /**
+   * Phone number
+   */
+  phone?: string | null;
+  /**
+   * Street 1
+   */
+  street1?: string | null;
+  /**
+   * Street 2
+   */
+  street2?: string | null;
+  /**
+   * City
+   */
+  city?: string | null;
+  /**
+   * State or province.
+   */
+  region?: string | null;
+  /**
+   * Zip or postal code.
+   */
+  postalCode?: string | null;
+  /**
+   * Country, 2-letter ISO 3166-1 alpha-2 code.
+   */
+  country?: string | null;
+
+};
+
+export type BillingInfo = {
+  lastName?: string | null;
```

Comment:
I looked at the backend code, and I believe this type is incorrect, e.g. `?` should be removed from each property.

Furthermore, I think the backend should be improved to avoid `undefined`, as JSON doesn't support `undefined`. Instead we should use `null` for `undefined` values.

These "minor" changes will make the typing work so much easier for consumers of this SDK/API.

Reply:
Apologies for not catching this in the backend PRs before.

Reply:
The problem is that this is how the Recurlt SDK defines it. We can add checks on API and always return null and make all properties required...

Reply:
Our backend always returns a proper structure where each property is always defined:
```
      return {
        lastName: billingInfo.lastName,
        firstName: billingInfo.firstName,
        company: billingInfo.company,
        address: billingInfo.address,
        paymentMethod: {
          cardType: billingInfo.paymentMethod?.cardType,
          firstSix: billingInfo.paymentMethod?.firstSix,
          lastTwo: billingInfo.paymentMethod?.lastTwo,
          expMonth: billingInfo.paymentMethod?.expMonth,
          expYear: billingInfo.paymentMethod?.expYear
        }
      };
```

.. therefore those question marks are wrong.

The types for _each property_  can be whateer Recurly SDK defines for them.

Reply:
Will update, thanks

Reply:
Updating also on API here https://github.com/lensapp/lenscloud/pull/1616